app/javascript/components/physical-storage-form/physical-storage-form.schema.js

```diff
@@ -0,0 +1,75 @@
+import { componentTypes, validatorTypes } from '@@ddf';
+
+function createSchema(data) {
+  return {
+    fields: [
+      {
+        component: componentTypes.SELECT,
+        name: 'ems_id',
+        id: 'ems_id',
+        label: __('Provider:'),
+        options: data.provider.options,
+        onChange: data.provider.onChange,
+        validate: [{
+          type: validatorTypes.REQUIRED,
+          message: __('Required'),
+        }],
+      },
+      {
+        component: componentTypes.TEXT_FIELD,
+        name: 'name',
+        id: 'name',
+        label: __('Name:'),
+        validate: [{
+          type: validatorTypes.REQUIRED,
+          message: __('Required'),
+        }],
+      },
+      {
+        component: componentTypes.SELECT,
+        name: 'physical_storage_family_id',
+        id: 'physical_storage_family_id',
+        label: __('System Type:'),
+        options: data.systemTypes.options,
```

Comment:
I would also do this via `loadOptions` that would build a request based on the passed `ems_id` and also render it only when that field is set:
```js
condition: {
  when: 'ems_id',
  isNotEmpty: true,
}
``